pxcentral-operator-to-chart-upgrade/README.md

```diff
@@ -0,0 +1,31 @@
+## Steps to upgrade operator based onprem central deployment to helm chart.
+
+1. Create volumesnapshot from existing onprem central pvc's in the same namespace.
+```console
+$ kubectl apply -f create_volume_snapshot_from_pvc.yaml
+```
+
+2. Verfiy volume snapshot for all pvc's are available in the same namespace.
+```console
+$ kubectl get volumesnapshot --namespace portworx
```

Comment:
Use storkctl here, it'll show the status of the snapshot

Reply:
Changed.